sqlglot/parser.py

```diff
@@ -551,13 +553,14 @@ class Parser(metaclass=_Parser):
 
     MODIFIABLES = (exp.Subquery, exp.Subqueryable, exp.Table)
 
-    CREATABLES = {
-        TokenType.TABLE,
-        TokenType.VIEW,
+    CREATEABLE = {
```

Comment:
why was the spelling here changed? i don't think createable is a word, https://www.thefreedictionary.com/creatable

Reply:
You're right, my bad. I should've checked -- I thought it's the same as "manageable" and changed it on a whim.
Fixed.